packages/rich-text/src/plugins/Hr/index.tsx

```diff
@@ -80,20 +81,22 @@ export function ToolbarHrButton(props: ToolbarHrButtonProps) {
 
     const hr = {
       type: BLOCKS.HR,
+      data: {},
       children: [{ text: '' }],
       isVoid: true,
     };
 
     const paragraph = {
       type: BLOCKS.PARAGRAPH,
+      data: {},
       children: [{ text: '' }],
     };
 
     const hasText = !!getText(editor, editor.selection.focus.path);
 
-    hasText ? insertNodes(editor, hr) : setNodes(editor, hr);
+    hasText ? Transforms.insertNodes(editor, hr) : setNodes(editor, hr);
 
-    insertNodes(editor, paragraph);
+    Transforms.insertNodes(editor, paragraph);
```

Comment:
TypeScript got mad at me for not using Slate's `Transforms.insertNodes` after the refactor but trust me that it's doing the same thing as udecode's implementation was before.